2-mandatory/3-groceries.js

```diff
@@ -27,13 +27,31 @@ Exercise 1:
 */
 // Gather all week item names into this array
 let weeklyGroceriesToBuy = [];
+for (const key in weeklyMealPlan)
+{
+    weeklyMealPlan[key].forEach(ingredient =>
+    {
+        if(!weeklyGroceriesToBuy.includes(ingredient))
+        {
+            weeklyGroceriesToBuy.push(ingredient);
+        }
+    })
+}
```

Comment:
Code is not indented properly, making it harder to read

Reply:
I pushed the code properly indented